modules/client-configuration/pages/autoinst-cdrom.adoc

```diff
@@ -30,11 +30,13 @@ In order to prepare an ISO image with Cobbler, you need to prepare a distributio
 For information about creating a distribution, see xref:client-configuration:autoinst-distributions.adoc[Autoinstallable Distributions].
 For information about creating a profile, see xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
 
-The Cobbler [command]``buildiso`` command takes parameters to define the name and output location of the boot ISO.
+The Cobbler [command]``buildiso`` command takes parameters to define the name and output location of the boot ISO. A distribution
+has to be provided in all cases with [option]``--distro``. 
+
```

Comment:
```suggestion
The Cobbler [command]``buildiso`` command takes parameters to define the name and output location of the boot ISO.
Specifying the distribution with [option]``--distro`` is mandatory.
```

Reply:
For your background info: We (unfortunately) have 1-sentence-per-line policy.

Reply:
I somehow did not find that on the [wiki](https://github.com/uyuni-project/uyuni-docs/wiki/wug) but good to know. Thank you @keichwa.